includes/User.php

```diff
@@ -2062,9 +2062,25 @@ public function deleteCache() {
 		$this->load();
 		$wgMemc->delete( $this->getCacheKey() );
 		$wgMemc->delete( self::getCacheKeyByName( $this->getName() ) ); // SUS-2945
+		$wgMemc->delete( self::getUserIdentityBoxCacheKey( $this->getId() ) );
+		$wgMemc->delete( self::getLookupContribsCoreCacheKey( $this->getId() ) );
 		$this->userPreferences()->deleteFromCache( $this->getId() );
 	}
 
+	/**
+	 * Returns memcached key for UserIdentityBox.
+	 */
+	public static function getUserIdentityBoxCacheKey( int $userId ): string {
```

Comment:
how about to make it non-static...just regular public function ? will that work ? and than you can easily call it like `$user->getUserIdentityBoxCacheKey()` without passing `userId`